### PR TITLE
Handle different deliveryChannels

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Add support for an order that contains items that will be delivered and items that will be picked-up at a pick-up point.

#### What problem is this solving?
Better coverage of possible checkout scenarios.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/OrderInfo/OrderInfo.js [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
-      orderValue={data.value}
-    />
-  </div>
-)
+  )
+}
 
 OrderInfo.propTypes = {
   data: PropTypes.object.isRequired,
   profile: PropTypes.object.isRequired,
```
Comment (salesfelipe) [inline react/components/OrderInfo/OrderInfo.js]: Try to describe a proptype for this

Comment (victorhmp) [reply react/components/OrderInfo/OrderInfo.js]: Done

File: react/components/OrderInfo/Shipping/Shipping.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+import ProductList from '../ProductList'
 
-const Shipping = ({ data }) => (
-  <div className="mv8 flex flex-column justify-between">
-    <ShippingHeader shippingData={data.shippingData} />
-    <ShippingProducts products={data.items} currency={data.storePreferencesData.currencyCode} />
-  </div>
-)
+const Shipping = ({ data, currency }) => {
+  data = data.filter(parcel => (parcel.deliveryChannel === 'delivery'))
```
Comment (klzns) [inline react/components/OrderInfo/Shipping/Shipping.js]: Fazendo mutaçao na prop, atribuir a outra variavel

File: react/components/OrderInfo/StorePickUp/StorePickUp.js [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
 import StorePickUpHeader from './StorePickUpHeader'
+import ProductList from '../ProductList'
 
-const StorePickUp = ({ data }) => (
-  <div className="mv8 flex flex-column justify-between">
-    <StorePickUpHeader shippingData={data.shippingData} />
-  </div>
-)
+const StorePickUp = ({ data, currency }) => {
+  data = data.filter(parcel => (parcel.deliveryChannel === 'pickup-in-point'))
```
Comment (klzns) [inline react/components/OrderInfo/StorePickUp/StorePickUp.js]: idem

**Pull request merged**